articles/azure-monitor/containers/container-insights-log-search.md

```diff
@@ -96,6 +96,28 @@ InsightsMetrics
 | sort by TimeGenerated asc<br> &#124; project RequestsPerMinute = Val - prev(Val), TimeGenerated
 | render barchart 
 ```
+### Pods by name and namespace
```

Comment:
When i run this query i get timegenerated and loglne for a specified pod name in a namespace. Is that whats intended to do ? title above says pods by name & namespace,,,